src/agents/policy-manager.ts

```diff
@@ -44,6 +45,33 @@ export class PolicyManagerAgent {
     return tx;
   }
 
+  public static async policyDisabled(
+    transactingPower: TransactingPower,
```

Comment:
Considering that this is a read-only function, why is `transactingPower` needed here?